.github/workflows/unsafe_pr_checks.yml

```diff
@@ -0,0 +1,20 @@
+name: CI Checks - Unsafe
+on:
+  push:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  Integration-Tests-External:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'run unsafe ci checks')
```

Comment:
This condition was a bit confusing to me on first pass, so to break down for anyone else:

`github.event.pull_request.labels.*.name` this statement is applying a filter on the `github.event.pull_request.labels` var, pull out a list of label names, e.g. `["safe to run", "dont merge", ...]`.

`contains(list, "value")` is then check if `"value"` is an element of `list`.

It seems simple when you put it that way, but if you don't know about the filter stage, it could look as though `github.event.pull_request.labels.*.name` evaluates to a string and we're checking if that string contains the PR name (which would be less robust).

https://docs.github.com/en/actions/learn-github-actions/expressions#object-filters